### PR TITLE
Remove fixListIndentation() filter.

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Remove the fixListIndentation() filter in favor of list converters fix. See ckeditor/ckeditor5-list#115.

---

### Additional information

* Sub PR for: https://github.com/ckeditor/ckeditor5-list/pull/141

### CODE REVIEW

File: tests/data/normalization.js [hunk 1]
```diff
@@ -57,13 +57,6 @@ describe( 'PasteFromOffice - normalization', () => {
 		editorConfig
 	} );
 
-	generateTests( {
```
Comment (Reinmar) [inline tests/data/normalization.js]: Shouldn't we remove something from `tests/_data/paste-from-google-docs/lists/index.js` too?

Comment (jodator) [reply tests/data/normalization.js]: Oh, I'll check.

Comment (jodator) [reply tests/data/normalization.js]: I've removed the google docs lists tests.

Comment (Reinmar) [reply tests/data/normalization.js]: Just to be sure – we still have an integration test for GDocs lists here?

Comment (jodator) [reply tests/data/normalization.js]: I've updated the PR with only normalization fixtures removed.

The fixtures/tests for integration were not touched.

**Pull request merged**